lib/active_merchant/billing/gateways/cyber_source_rest.rb

```diff
@@ -129,8 +156,17 @@ def add_address(post, payment_method, address, options, address_type)
         }.compact
       end
 
+      def add_merchant_description(post, options)
+        return unless options[:merchant_descriptor_name] || options[:merchant_descriptor_address1] || options[:merchant_descriptor_locality]
+
+        merchant = post[:merchantInformation][:merchantDescriptor] = {}
+        merchant[:name] = options[:merchant_descriptor_name] if options[:merchant_descriptor_name]
+        merchant[:address1] = options[:merchant_descriptor_address1] if options[:merchant_descriptor_address1]
+        merchant[:locality] = options[:merchant_descriptor_locality] if options[:merchant_descriptor_locality]
+      end
+
       def url(action)
-        "#{(test? ? test_url : live_url)}#{action}"
+        "#{(test? ? test_url : live_url)}/pts/v2/#{action}"
```

Comment:
Do we expect to need to hit a different endpoiont for other types of actions later? If not, I think we can just add it to the base `self.live/test_url`s

Reply:
Yes, reviewing the changes from @scastilloendava looks like the paths for [store/unstore](https://github.com/activemerchant/active_merchant/pull/4709) have another subpath